src/Gui/TaskMeasure.cpp

```diff
@@ -149,6 +151,54 @@ void TaskMeasure::setMeasureObject(Measure::MeasureBase* obj) {
 }
 
 
+App::DocumentObject* TaskMeasure::createObject(std::string& measureClass) {
+    auto type = Base::Type::fromName(measureClass.c_str());
+
+    if (type == Base::Type::badType()) {
+        return nullptr;
+    }
+
+    auto measureObj = static_cast<App::DocumentObject*> (type.createInstance());
```

Comment:
This is unsafe code. There is no guarantee that measureClass is of App::DocumentObject and if not this cast causes undefined behaviour.